src/databricks/labs/ucx/install.py

```diff
@@ -216,9 +216,21 @@ def _run_configured(self):
         logger.info(msg)
 
     def _create_database(self):
-        if self._sql_backend is None:
-            self._sql_backend = StatementExecutionBackend(self._ws, self.current_config.warehouse_id)
-        deploy_schema(self._sql_backend, self.current_config.inventory_database)
+        try:
+            if self._sql_backend is None:
+                self._sql_backend = StatementExecutionBackend(self._ws, self.current_config.warehouse_id)
+            deploy_schema(self._sql_backend, self.current_config.inventory_database)
+        except BadRequest as ex:
+            if "UNRESOLVED_COLUMN.WITH_SUGGESTION" in str(ex):
```

Comment:
the goal is is to raise a more informative exception - not just do a naive log, not to swallow it (which is even worse).

Reply:
Now if this exception happens it will error out with proper message.